### DISCUSSION TITLE
WHAT IS THE BEST VPS SERVER TO USE FOR POSTAL

### DISCUSSION THREAD

**Discussion by Itseric100** [postalserver/postal#2259] [Help with using Postal]
Alright, i have actually explored a lot of different vps provider such as contabo, hostwind, hostsailor, namecheap amongst others and i found they probably have different policies. Also, email isp also probably have vps providers they are more compatible with or work better with that makes your email deliverability more assured. For example, i tried to send a mail to OUTLOOK from my postal on vps from hostsailor which had a soft fail with an error that some of hostsailor components are blocked by msn and wont make the email go through.
     Here is my question please, people that send thousands of email to outlook/hotmail/office and other microsoft isp, MAY I ASK WHICH VPS YOUR POSTAL IS BUILT ON? WHERE DO YOU PURCHASE YOUR VPS FROM? And what do you think is the best vps provider that is most efficient and accepted by most of the email isp. THANKS IN ADVANCE.

**Comment by willpower232** [comment]
Please don't open duplicate discussions.

There is no concrete list of "working" server providers as every company is different. Email sending can be based on reputation which does take time to build up so if you can find a server provider which allows you to send messages, it is unlikely you will be able to go straight into mass email sending. As you have seen, you may be able to get to the last step and find the whole provider is blocked so you have to start over.

Postal is intended for transactional emails rather than marketing so if you're relying on doing marketing then you may wish to find a more established email marketing company.

We are using AWS which we have been for several years so don't have many issues emailing Microsoft users.

**Reply by Itseric100** [reply]
Thank you so much
